_posts/blog/2018-09-05/2018-09-05-troubleshooting-and-how-to-help-with-development-on-openbazaar.markdown

```diff
@@ -0,0 +1,110 @@
+
+### Researching Resources
+
+For lack of a solution already existing, getting your unreported problem solved will be signficantly more successful if you can get the report into the right hands. We have quite a few places across the internet where the OpenBazaar community offers help to each other. Take advantage of the open community by sharing your challenge in the appropriate channel.
+
+Let's quickly walk through the **common resources** available and how they may be useful.
+
+**1) [Frequently Asked Questions and Knowledge Base](https://openbazaar.zendesk.com/)**
+
+Here we keep instructions for common tasks such as software setup, establishing your store, and handling moderation tasks. There’s also information about features which are unique to OpenBazaar like our multi-signature escrow or how vacation mode is used. This is primarily a read-only site which will surface the most regular questions from our users.
+
+**2) Active Updates on [Twitter](https://twitter.com/openbazaar)**
+
+Twitter can be a good source of up-to-the-minute news as well as a way to reach out to our team for urgent requests. Here you can find dialogue with fans, critics, news, updates, and promotions from our marketing team. It is an open platform for discussion with a short-form approach to dialogue.
+
+**3) Community Discussion on [Reddit](https://reddit.com/r/openbazaar)**
+
+Reddit allows for topical, threaded discussions and lets users discuss all sorts of things. Some leading topics in the OpenBazaar subreddit include: new stores, decentralized ecommerce news, cryptocurrency mechanics and issues which are hard to track down. The discussion on Reddit shifts over time but its persistence gives a good archive of past issues and thought-provoking conversation. Participating in the OpenBazaar subreddit can be anonymous and has no barrier to posting a question or comment short of creating a Reddit account. Most anything even slightly related to OpenBazaar can be posted here.
+
+**4) Real-time Discussion on [Slack](https://openbazaar.org/slack) and [Telegram](https://t.me/OpenBazaarGroup)**
+
+For high-bandwidth conversation with other users and contributors, our team and community congregate in these 2 discussion forums that function a lot like real-time chatrooms. Some areas of these communities focus on 3rd party search providers, moderators and escrow, new ideas being tested on OpenBazaar, as well as offering each other help and support on common issues. Slack, byfar the most popular of the bunch, has the most regular activity as OB1 and many other developers are engaged there. Telegram is not quite as active but is a nice addition to Slack were people can pop in to ask quick questions about OpenBazaar or determine their next steps troubleshooting an issue.
```

Comment:
Insert an "and" between escrow and new ideas.

"focus on ... moderators and escrow, and new ideas..."

Reply:
Put a space in "by far".

Reply:
Nitpick, two isn't a bunch. ;)

"... the more popular of the two."

Reply:
Add a comma in "most regular activy, as OB..."

Reply:
> as OB1 and many other developers are engaged there.

This refers to OB1 as a singular developer.

Maybe "the OB1 team and many other developers..."